rkt/trust.go

```diff
@@ -76,9 +76,12 @@ func runTrust(args []string) (exit int) {
 		return 1
 	}
 
-	// if the user included a scheme with the prefix, discard it
-	flagPrefix = strings.TrimPrefix(flagPrefix, "http://")
-	flagPrefix = strings.TrimPrefix(flagPrefix, "https://")
+	// if the user included a scheme with the prefix, error on it
+	u, err := url.Parse(flagPrefix)
```

Comment:
there's supposed to be && u.Scheme != "", this was an experiment I forgot to undo while playing with error messages.